plugins/woocommerce/src/Internal/Admin/Marketing/MarketingSpecs.php

```diff
@@ -136,34 +136,41 @@ protected function is_marketing_channel_plugin( array $plugin_data ): bool {
 	/**
 	 * Load knowledge base posts from WooCommerce.com
 	 *
-	 * @param string|null $category Category of posts to retrieve.
+	 * @param string|null $term Term of posts to retrieve.
 	 * @return array
 	 */
-	public function get_knowledge_base_posts( ?string $category ): array {
-		$kb_transient = self::KNOWLEDGE_BASE_TRANSIENT;
-
-		$categories = array(
-			'marketing' => 1744,
-			'coupons'   => 25202,
+	public function get_knowledge_base_posts( ?string $term ): array {
+		$terms = array(
+			'marketing' => array(
+				'taxonomy' => 'category',
+				'term_id'  => 1744,
+				'argument' => 'categories',
+			),
+			'coupons'   => array(
+				'taxonomy' => 'post_tag',
+				'term_id'  => 1377,
+				'argument' => 'tags',
+			),
 		);
```

Comment:
I think we don't need the `taxonomy` field because it is not being used?

Reply:
Thank you, @gan, for taking the time to review this PR. The `taxonomy` field is there for readability purposes because `marketing` and `coupons` are different terms belonging to different taxonomies. Alternatively, we can move it to a comment. What do you think?

Reply:
> The `taxonomy` field is there for readability purposes because `marketing` and `coupons` are different terms belonging to different taxonomies.

Is that a standard or common way to do that in WordPress? If it is, then I guess it's fine (and that would be a TIL for me too). 

Otherwise, I'm leaning towards moving it to a comment, because having something like `'taxonomy' => 'post_tag'` made me wonder, e.g. "where is the `'post_tag'` value being shown or used?".

Reply:
I did a bit more reading and I think now I understand that terms are "items within a taxonomy", and taxonomy here can be either a category or a tag. So I guess it's fine to leave the taxonomy field there, it was just me not familiar with the usage. 😄 

References:

- https://developer.wordpress.org/themes/basics/categories-tags-custom-taxonomies/#terms
- https://developer.wordpress.org/plugins/taxonomies/working-with-custom-taxonomies/

Reply:
Thank you, @gan, for your feedback. I'll merge this PR.